editor/editor_properties.cpp

```diff
@@ -3302,12 +3302,11 @@ void EditorPropertyResource::update_property() {
 				sub_inspector->set_read_only(is_read_only());
 				sub_inspector->set_use_folding(is_using_folding());
 
-				sub_inspector->set_draw_focus_border(false);
```

Comment:
This leads to double borders.
![image](https://github.com/user-attachments/assets/686b9eb5-2e7a-4b44-85ab-81b9530d372d)

Reply:
Forget that it will also draw when the child control has focus. Has been modified.